.github/workflows/validate-json.yaml

```
@@ -18,18 +18,53 @@ jobs:
         with:
           use-public-rspm: true
 
-      - name: Install V8
-        run: sudo apt install libv8-dev
+      - uses: r-lib/actions/setup-r-dependencies@v2
+        with:
+          extra-packages:
```

```suggestion
          packages:
```

Since you don't need to install epiparameter or any of its dependencies